knowledge-base/how-to-restrict-export-formats-in-preview.md

```diff
@@ -0,0 +1,53 @@
+---
+title: How to restrict export formats in Report Preview
+description: How to limit the rendering extensions in the Report Preview mode of the Report Server
+type: how-to
+page_title: How to restrict export formats
+slug: how-to-restrict-export-formats-in-preview
+position: 
+tags: 
+ticketid: 1408256
+res_type: kb
+---
+
+## Environment
+<table>
+    <tbody>
+	    <tr>
+	    	<td>Product</td>
+	    	<td>Progress® Telerik® Report Server</td>
+	    </tr>
+    </tbody>
+</table>
+
+
+## Description
+I want to hide some of the rendering extensions.
```

Comment:
This description is very short. The description in the meta information is better.

Reply:
Agree :)